pkgs/by-name/li/librespeed-rust/package.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+let
+  # https://github.com/librespeed/speedtest-rust/pull/7
+  version = "unstable-2024-09-28";
+  src = fetchFromGitHub {
+    owner = "librespeed";
+    repo = "speedtest-rust";
+    rev = "a74f25d07da3eb665ce806e015c537264f7254c9";
+    hash = "sha256-+G1DFHQONXXg/5apSBlBkRvuLT4qCJaeFnQSLWt0CD0=";
+  };
+in
+rustPlatform.buildRustPackage {
+  pname = "librespeed-rust";
+  inherit version src;
+
+  cargoLock.lockFile = "${src}/Cargo.lock";
+
+  # error: linker `aarch64-linux-gnu-gcc` not found
+  postPatch = ''
+    rm .cargo/config.toml
+  '';
+
+  postInstall = ''
+    cp -r assets $out/
+  '';
+
+  meta = {
+    description = "A very lightweight speed test implementation in Rust.";
```

Comment:
This is a silly nixpkgs thing but descriptions aren't supposed to end in `.`  https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes